torch/csrc/jit/python/init.cpp

```diff
@@ -488,6 +488,11 @@ void initJITBindings(PyObject* module) {
       .def(
           "_jit_pass_insert_prepacked_ops",
           [](script::Module& module) { return insertPrePackedOps(module); })
+      .def(
+          "_jit_pass_fuse_relu_w_prepacked_linear_conv",
+          [](script::Module& module) {
+            return fusePrePackedLinearConvWithRelu(module);
```

Comment:
Do we also support different variations of ReLU6 and Hardtanh in this rule?

Reply:
I did not add that as part of this, but given the patters are just string. We can replace relu with hardtanh. So should be doable.

Reply:
Can you please enable ReLU6 as well (if not already part of this PR.)  Reason being that it is used in MobileNetV2 and I am sure it won't be long before we receive a usecase in which that is causing a performance issue.

Reply:
Sure. Will do this in this PR itself. Note that ReLU6 in pytorch just maps to hardtanh.